code/game/objects/structures/fluff.dm

```diff
@@ -262,34 +262,6 @@
 	desc = "A pile of scrap metal. It seems damaged beyond repair."
 	icon_state = "clockgolem_dead"
 
-/obj/structure/fluff/hedge
-	name = "hedge"
-	desc = "A large bushy hedge."
-	icon = 'icons/obj/smooth_structures/hedge.dmi'
-	icon_state = "hedge-0"
-	base_icon_state = "hedge"
-	smoothing_flags = SMOOTH_BITMASK
-	smoothing_groups = list(SMOOTH_GROUP_HEDGE_FLUFF)
-	canSmoothWith = list(SMOOTH_GROUP_HEDGE_FLUFF)
-	density = TRUE
-	anchored = TRUE
-	deconstructible = FALSE
-	max_integrity = 80
-
-/obj/structure/fluff/hedge/attacked_by(obj/item/I, mob/living/user)
-	if(opacity && HAS_TRAIT(user, TRAIT_BONSAI) && I.get_sharpness())
-		to_chat(user,span_notice("You start trimming \the [src]."))
-		if(do_after(user, 3 SECONDS,target=src))
-			to_chat(user,span_notice("You finish trimming \the [src]."))
-			opacity = FALSE
-	else
-		return ..()
-/**
- * useful for mazes and such
- */
-/obj/structure/fluff/hedge/opaque
-	opacity = TRUE
-
```

Comment:
Change not mentioned in PR body

Reply:
Fluff.dm ironically enough is just a thinly veiled miscellaneous file, i have moved them to where they will be found